packages/swing-store/src/transcriptStore.js

```diff
@@ -199,13 +202,13 @@ export function makeTranscriptStore(
    */
   function initTranscript(vatID) {
     ensureTxn();
-    sqlWriteSpan.run(vatID, 0, 0, initialHash, 1);
-    const newRec = spanRec(vatID, 0, 0, initialHash, 1);
+    sqlWriteSpan.run(vatID, 0, 0, initialHash, 1, 0);
```

Comment:
nit: to make the code more readable, I might do `const incarnation = 0` and then `run(... incarnation)`, and likewise for the other `0` and `1` which have semantics. Otherwise you have to search back to find the `sqlWriteSpan` definition (ok ok its only ten lines earlier) to see what the positional parameters mean.

OTOH, if that gave `prettier` an excuse to wrap the line, I'd hate it. So I'd probably find short names for the four integers there.